REFERENCE.md

```diff
@@ -283,13 +283,14 @@ Data type:
 ```puppet
 Variant[Pattern[/^file:\/\//],
     Pattern[/^puppet:\/\//],
-    Stdlib::Httpsurl, Stdlib::Httpurl
+    Stdlib::Httpsurl, Stdlib::Httpurl,
+    Undef
   ]
 ```
 
 Source of Wildfly tarball installer.
```

Comment:
Can you adjust this to describe the new behavior?  I think about something like "Custom source of Wildfly tarball installer (automatic when undef)"

Reply:
Good catch. Changed to "Custom source of Wildfly tarball installer. Defaults to standard Wildfly repositories.", so default behaviour is clear.